packages/crypto/src/bip39.ts

```diff
@@ -1,8 +1,5 @@
-import { fromHex, toHex, toUtf8 } from "@cosmjs/encoding";
+import { fromHex, toHex } from "@cosmjs/encoding";
 import * as bip39 from "bip39";
-import { pbkdf2 } from "pbkdf2";
-import * as unorm from "unorm";
-
```

Comment:
Lint is complaining because imports should be in two blocks separated by an empty line: external and local.